src/styles/global.css

```diff
@@ -37,3 +37,29 @@ body {
     display: block;
   }
 }
+
+article {
+  /* Support mobile devices without scaling if a component exceeds the article
+     * width, e.g., for tables 
+    */
+  overflow-x: scroll;
+}
+
+/* For tables, use a fixed table layout so a page that features multiple tables
+ * with the same columns can display all columns with the same alignment.
+ * Specify both a cell width and a table width so the columns are always
+ * aligned. If there are fewer than three cells per row, use a wider cell.
+ If there are more than three cells per row, use a narrower cell */
+table {
+  table-layout: fixed;
+  width: 675px;
+  max-width: 675px;
+}
+
+table:not(table:has(td:nth-of-type(3))) td {
+  width: 300px;
+}
+
+table:has(td:nth-of-type(3)) td {
+  width: 150px;
```

Comment:
Maybe instead of a fixed width, just do 25% width, with a flexible width (instead of fixed/max-width on lines 55 and 56)?

Also, it could aid in legibility to make 2nd-4th columns center-aligned (with first column still left-aligned)
![image](https://github.com/user-attachments/assets/1b24b2c7-16a7-4e19-a2cb-7b808b34c69b)

Reply:
That works for me! A 25% width would (as far as I understand it) behave as expected for four-column tables, so we could define separate `table:has` blocks for different column sizes, each with a different percentage, if that makes sense. 

It looks like the widest table in the docs has six columns, the one in [this section](https://goteleport.com/docs/admin-guides/api/access-plugin/#create-a-google-sheets-spreadsheet), which only has a single row. There are a few tables with five columns, etc., [this section](https://goteleport.com/docs/reference/helm-reference/teleport-cluster/#proxyprotocol).

Reply:
I think that for most tables, flexible column widths are probably good. This page is the only one I know of where the tables are all related, and so it makes sense to standardize column widths so they align across the tables. If it's easy to apply a class to all of these tables, you could do it that way, too.

Reply:
👍 We could define a class that applies this style. In terms of implementing the style for just the tables in the feature matrix, it's not possible to apply a style to a table using Markdown table syntax (the way the tables are [currently defined](https://github.com/gravitational/teleport/blob/master/docs/pages/includes/edition-comparison.mdx?plain=1)). Instead, we could either:
- Use raw HTML for the table, adding the class to that
- Wrap the Markdown table in a `<div>` with the class
  
My personal vote would be to wrap the Markdown table in a `div`, which avoids the distracting HTML table tags, and would be quick to execute. What do you think?

Reply:
wrapping it in a div sounds like the less complex way to me as well, but I don't have strong opinions on implementation details as long as the resulting UX is good. Thanks!